recipes/nest/activate.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+NEST_INSTALL_DIR=$PREFIX
```

Comment:
Are these variables actually needed? From a quick glance at the NEST source it looks like it automatically uses the correct locations.

Reply:
Yes

Reply:
I experimented a lot and finally removed PYTHONPATH and PATH.